### PR TITLE
fix(): add optional chaining everywhere a profile object is used

### PR DESCRIPTION
**Pull request by didd**
## Description

- add optional chaining everywhere a profile object is used

## Checklist

- [X] I have read the **README** document
- [X] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [X] All new and existing tests passed
- [X] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: libs/design-system-components/src/components/ExtensionEditStep3Form/Collaborators/index.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const Collaborators: React.FC<CollaboratorsProps> = ({
   };
 
   const handleRemoveContributor = (profile: AkashaProfile) => {
-    setAddedContributors(prev => prev.filter(addedProfile => profile.id !== addedProfile?.id));
+    setAddedContributors(prev => prev.filter(addedProfile => profile?.id !== addedProfile?.id));
```
Comment (kenshyx) [inline libs/design-system-components/src/components/ExtensionEditStep3Form/Collaborators/index.tsx]: this one is not optional

Comment (kenshyx) [reply libs/design-system-components/src/components/ExtensionEditStep3Form/Collaborators/index.tsx]: or if it's optional, couldn't this lead to false positive?

Comment (didd) [reply libs/design-system-components/src/components/ExtensionEditStep3Form/Collaborators/index.tsx:66]: I made it optional on the param. From what I see in the code there is no guarantee the profile param will not be null.

Comment (didd) [reply libs/design-system-components/src/components/ExtensionEditStep3Form/Collaborators/index.tsx]: I thought more about this, yeah I have scraped the change. It doesn't actually make sense to make the param optional here.

**Review by kenshyx — APPROVED** [review]

**Pull request merged**